e2e/pages/web-pages/campaigns/campaigns.page.ts

```diff
@@ -0,0 +1,124 @@
+import { Page } from '@playwright/test';
+import { LanguagesEnum } from '../../../data/enums/languages.enum';
+import { bgLocalizationCampaigns, enLocalizationCampaigns } from '../../../data/localization';
+import { HomePage } from '../home.page';
+
+
+export class CampaignsPage extends HomePage {
+
+    constructor(page: Page) {
+        super(page);
+    }
+
+    private readonly donationGrid = ".InlineDonation-inlineDonationWrapper";
+    private readonly donationSupportButton = this.donationGrid + " a button";
+    private readonly filterButtonsCommonSelector = "ul button.CampaignFilter-filterButtons";
+    // private readonly campaignContainerItem = ".MuiGrid-container .MuiGrid-item";
+    private readonly cardActions = ".MuiCardActions-root";
+    private readonly cardActionButtons = this.cardActions + " button";
+    // Focused campaigns for tests (hardcoded text here because I didn't find it anywhere else)
+    protected readonly campaignSchoolVarnaText = "Училище за деца с нарушено зрение гр. Варна - стая за ерготерапия";
+    protected readonly campaignCrisisCenterText = "Кризисен център за пострадали от насилие - шанс за нов живот";
+    // Main headings
+    private readonly bgMainCampaignsHeading = bgLocalizationCampaigns.campaigns;
+    private readonly enMainCampaignsHeading = enLocalizationCampaigns.campaigns;
+    private readonly bgSupportCauseTodayHeading = bgLocalizationCampaigns.cta['support-cause-today'];
+    private readonly enSupportCauseTodayHeading = enLocalizationCampaigns.cta['support-cause-today'];
+    private readonly bgSupportNowActionButtonText = bgLocalizationCampaigns.cta['support-now'];
+    private readonly enSupportNowActionButtonText = enLocalizationCampaigns.cta['support-now'];
```

Comment:
Isn't there a way to use the t() function?
Hardcoding those seems like a lot of code duplication to me.